src/enochecker/post.html

```diff
@@ -458,11 +470,11 @@ <h3> Responses </h3>
 
           checker_request_count++
           checker_pending_requests--
-          update_pending();
+          update_pending()
         }
 
-        xhr.send(request_json);
-        checker_pending_requests++;
+        xhr.send(request_json)
```

Comment:
Any particular reason why you removed all these semi-colons?^^

Reply:
To make domenukk happy